package.json

```diff
@@ -28,6 +28,7 @@
     "date-fns": "^2.12.0",
     "dayjs": "^1.8.35",
     "esm": "^3.2.25",
+    "emoji-mart": "^3.0.1",
```

Comment:
while testing with env/ohbm and bunch of other perspective PRs merged, https://github.com/yarikoptic/sparkle/pull/new/my-ohbm  915857eb518a529ac8b00ce26c9fa5ea54ce8cd4  I noticed that the text in emoji-mart selector (which has white background -- easy to change to black @soichih ??) was white.

Reply:
Just fixed it and PR-ed to @margulies:feature/markdown

Reply:
Thank you, merged and here

Reply:
Awesome!

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1383/commits/1a52db1fc3b7cd24aef71c067815aac48508fe18

Reply:
@soichih FYI: We have a `z()` `z-index` layer helper in `scss/constants` that we use for all of our z-index`'s, rather than defining it directly in individual files, so we'll need to get that fixed up from that patch.

Reply:
colors were fixed and z-index also: https://github.com/sparkletown/sparkle/pull/1383/files#diff-e81ecc2f161d11a623a02514686e44a4a4b3fb7a08a5c3b3fecab9895652ab95R135  . I think this could be marked "resolved"